steam_wrapper/steam_wrapper.py

```diff
@@ -125,7 +126,18 @@ def check_allowed_to_run(current_info):
         shutil.rmtree(steam_home)
 
 
-def migrate_config():
+def make_xdg_ignores(current_info, xdg_name, path):
+    ignores = []
+    filesystems = current_info["filesystems"]
+    for filesystem in filesystems:
+        filesystem_path = filesystem.split(":")[0]
+        if filesystem_path.startswith(xdg_name):
+            ignores.append(re.sub(f"^{xdg_name}", path,
+                                  filesystem_path))
```

Comment:
I've meant something like this when suggested `os.path.split()` above (didn't actually test it).
```suggestion
        filesystem_path = filesystem.rsplit(':', 1)[0]
        path_seq = os.path.split(filesystem_path)
        if path_seq[0] == xdg_name:
            ignores.append(os.path.join(*path_seq[1:]))
```